perception/multi_object_tracker/src/tracker/model/big_vehicle_tracker.cpp

```diff
@@ -249,29 +255,36 @@ bool BigVehicleTracker::measureWithPose(
       enable_velocity_measurement = true;
     }
   }
+
   // pos x, pos y, yaw, vx depending on pose measurement
   const int dim_y = enable_velocity_measurement ? 4 : 3;
-  double measurement_yaw = tier4_autoware_utils::normalizeRadian(
-    tf2::getYaw(object.kinematics.pose_with_covariance.pose.orientation));
-  {
-    Eigen::MatrixXd X_t(ekf_params_.dim_x, 1);
-    ekf_.getX(X_t);
-    // Fixed measurement_yaw to be in the range of +-90 degrees of X_t(IDX::YAW)
-    while (M_PI_2 <= X_t(IDX::YAW) - measurement_yaw) {
-      measurement_yaw = measurement_yaw + M_PI;
-    }
-    while (M_PI_2 <= measurement_yaw - X_t(IDX::YAW)) {
-      measurement_yaw = measurement_yaw - M_PI;
-    }
+  double measurement_yaw = getMeasurementYaw(object);  // get sign-solved yaw angle
+  Eigen::MatrixXd X_t(ekf_params_.dim_x, 1);           // predicted state
+  ekf_.getX(X_t);
+
+  // save last input bounding box
+  if (last_input_bounding_box_.length == -1) {
+    last_input_bounding_box_ = {
+      object.shape.dimensions.x, object.shape.dimensions.y, object.shape.dimensions.z};
   }
 
+  /* get offset measurement*/
+  autoware_auto_perception_msgs::msg::DetectedObject offset_object;
+  utils::calcAnchorPointOffset(
+    last_input_bounding_box_.width, last_input_bounding_box_.length, last_nearest_corner_index_,
+    object, X_t(IDX::YAW), offset_object, tracking_offset_);
+  // std::cout << "MOT_offset: " << tracking_offset_.x() << " " << tracking_offset_.y()
+  //           << " closest_corner: " << last_nearest_corner_index_ << " "
+  //           << object.kinematics.pose_with_covariance.pose.position.x << " "
+  //           << object.kinematics.pose_with_covariance.pose.position.y << std::endl;
```

Comment:
can you remove this comments?

Reply:
Thaks. This is fixed in https://github.com/autowarefoundation/autoware.universe/commit/dbbbf3d8a18a39a5f6f78931e2c82ce6ff9fc875.